Client/Application/NavigationRouter.swift

```diff
@@ -210,6 +215,10 @@ enum NavigationPath {
         }
     }
 
+    private static func handleReferral(code: String, with bvc: BrowserViewController) {
```

Comment:
I feel this needs a bit of refactoring, I don't see a purpose for this method, the contents could be easily inlined on the caller removing some unnecessary complexity.

Reply:
removed :)